### ISSUE TITLE
Jitsi now start with a 'Join meeting' page

### ISSUE THREAD

**Issue by mroelandts** [HackerShackOfficial/Smart-Doorbell#4]
If you go to a meeting link you will now get a 'Join meeting' page instead of immediately joining the meeting.
This breaks the whole system as the raspberry does not joins the meeting without clicking the 'Join' button...

![107575648_275531767116633_1812312307624554403](https://user-images.githubusercontent.com/17315891/87011012-a474eb80-c1c7-11ea-839a-566f2feb7404.jpg)

**Comment by mroelandts** [comment]
It appears, there is now a "Don't show this again" checkbox.
after this has been checked, everything runs fine!

**Comment by dtony** [comment]
Just change line 100 from :
`        return "http://meet.jit.si/%s" % self.chat_id`
to : 
`        return "http://meet.jit.si/%s#config.prejoinPageEnabled=false" % self.chat_id`

**Comment by karsom** [comment]
> Just change line 100 from :
> ` return "http://meet.jit.si/%s" % self.chat_id`
> to :
> ` return "http://meet.jit.si/%s#config.prejoinPageEnabled=false" % self.chat_id`

Yep, this works, I have had this problem as my display is not touchscreen, and this fixes the only problem I had!!
tysm
